apps/meteor/tests/e2e/moderation.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { Users } from './fixtures/userStates';
+import { Moderation, HomeChannel } from './page-objects';
+import { createTargetChannel, deleteChannel } from './utils';
+import { test, expect } from './utils/test';
+
+test.use({ storageState: Users.admin.state });
+
+test.describe.serial('moderation-console', () => {
+	let poModeration: Moderation;
+	let poHomeChannel: HomeChannel;
+	let targetChannel: string;
+	const singleMessage = 'Message to report';
+
+	test.beforeEach(async ({ page }) => {
+		poHomeChannel = new HomeChannel(page);
+		poModeration = new Moderation(page);
+
+		await page.goto('/home');
+	});
+
+	test.beforeAll(async ({ api }) => {
+		targetChannel = await createTargetChannel(api);
+	});
+
+	test.beforeAll(async ({ browser }) => {
+		const user1Page = await browser.newPage({ storageState: Users.user1.state });
+		const user1Channel = new HomeChannel(user1Page);
+		await user1Page.goto(`/channel/${targetChannel}`);
+		await user1Channel.waitForChannel();
+		await user1Channel.content.sendMessage(singleMessage);
+		await expect(user1Channel.content.lastUserMessage).toContainText(singleMessage);
+
+		await user1Page.close();
+	});
+
+	test.afterAll(async ({ api }) => {
+		await deleteChannel(api, targetChannel);
+	});
+
+	test.describe('Message reporting', async () => {
+		test('should user be able to report a given message', async () => {
+			await poHomeChannel.sidenav.openChat(targetChannel);
+			await poHomeChannel.content.openLastMessageMenu();
+			await poModeration.reportMsgButton.click();
+			await poModeration.reportMessageReasonText.fill('Reason to report');
+			await poModeration.reportMessageReasonSubmit.click();
```

Comment:
we do still need a assertion here, otherwise we're not testing anything...

Reply:
what happens when an user reports a message successfully?

Reply:
so this should be only a step for the assertion you're doing below! mix them up, then